README.md

```diff
@@ -279,70 +344,37 @@ NOTE that this functionality works on incremental models that use partitioning:
 }}
 ```
 
-#### Materialization
-
-Adapter supports all materializations provided by dbt-core.
-
-- table materialization
+##### Snapshots
```

Comment:
Does Snapshot is an actual materialization (it's here under Materializations section)? It seem to be more like a separate/different abstraction?

Reply:
See https://github.com/dbt-labs/dbt-core/blob/a7ff003d4f74763f126e77a0fc9c55e7a3311b3d/core/dbt/include/global_project/macros/materializations/snapshots/snapshot.sql